cmd/kvdpa-cli/kvdpa-cli.go

```diff
@@ -72,6 +73,19 @@ func getAction(c *cli.Context) error {
 	return nil
 }
 
+func addAction(c *cli.Context) error {
+	if c.Args().Len() != 2 {
+		usage := fmt.Sprintf("Usage: %v %v %v \n", c.App.Name, c.Command.Name, c.Command.ArgsUsage)
```

Comment:
This is not very maintainable. You can use cli.ShowAppHelp(c) to show the auto-generated help.

Reply:
done